src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -336,6 +341,10 @@ public SwerveModuleState[] getModuleStates() {
    * @param isOpenLoop if open loop control should be used for the drive velocity
    */
   public void setChassisSpeeds(ChassisSpeeds chassisSpeeds, boolean isOpenLoop) {
+    if (!RobotBase.isReal()) {
```

Comment:
just a small note there is a "Robot.isSimulation()" I think might be slightly more reable than "!RobotBase.isReal()"